solutionAgent/webcrawler.py

```diff
@@ -17,9 +21,15 @@ def search(self, query):
             str: The first result returned by the search engine.
         """
         try:
-            return(self.tool.invoke({"query": query},search_depth="advanced"))
+            return(self.tool.search(query = query,include_domains = ['https://stackoverflow.com','https://github.com'],search_depth="advanced"))
             
         except Exception as e:
             print(e)
             return None
-        
\ No newline at end of file
+
+if __name__ == '__main__':
+    crawl = WebCrawler()
+    print(crawl.search("""AttributeError: module 'numpy' has no attribute 'object'.
```

Comment:
What is this hardcoded search for?

Reply:
Ahh, sorry I forgot to remove my testing code. Just removed it now.